### DISCUSSION TITLE
Audio offset in multiplayer lobby.

### DISCUSSION THREAD

**Discussion by LacticWhale** [ppy/osu#23965] [Q&A]
### Type

Performance

### Bug description

[Windows 11]

In the multiplayer lobby, about 1 second before the start of the map, there was a sharp drop in performance (fps and delay turned red). And at the start of the beatmap, there was a noticeable shift in the sound, the audio started playing before the beatmap.

Not really breaking.    

### Version

2023.617.0

### Logs

[network.log](https://github.com/ppy/osu/files/11781237/network.log)
[performance.log](https://github.com/ppy/osu/files/11781238/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11781239/runtime.log)

**Comment by peppy** [comment]
This sounds very weird. The game auto-syncs when things go more than ~30ms out of sync.

**Reply by LacticWhale** [reply]
I got in only once but it was very noticable offset of 1-2 seconds. 
I started to play with shadowplay to try to capture it.
